mne/io/eeglab/eeglab.py

```diff
@@ -104,11 +105,26 @@ def _eeg_has_montage_information(eeg):
     return has_pos
 
 
-def _get_eeg_montage_information(eeg, get_pos):
-
-    pos_ch_names, ch_names, pos = list(), list(), list()
+def _get_montage_information(eeg, get_pos):
+    """Get channel name, type and montage information from ['chanlocs']."""
+    ch_names, ch_types, pos_ch_names, pos = list(), list(), list(), list()
     for chanloc in eeg.chanlocs:
+        # channel name
         ch_names.append(chanloc['labels'])
+
+        # channel type
+        if 'type' in chanloc:
+            ch_type = chanloc['type']
+            if isinstance(ch_type, str) and 0 < len(ch_type):
+                ch_type = ch_type.strip().lower()
+                ch_types.append(ch_type if ch_type in _PICK_TYPES_KEYS
+                                else 'eeg')
+            else:
+                ch_types.append('eeg')  # guarantee same size as ch_names
+        else:
+            ch_types.append('eeg')  # guarantee same size as ch_names
```

Comment:
Instead of repeating a bunch of `ch_types.append` statements and using a lot of conditionals, it seems to be cleaner to say "assume EEG unless shown otherwise" like:
```suggestion
        ch_type = 'eeg'
        try_type = chanloc.get('type', None)
        if isinstance(try_type, str):
            try_type = try_type.strip().lower()
            if try_type in _PICK_TYPES_KEYS:
                ch_type = try_type
        ch_types.append(ch_type)
```